src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const tables = table.children;
+
+for (const part of tables) {
+  const linescollection = part.children;
+
+  for (const line of linescollection) {
+    let newElement;
+
+    if (part.tagName === 'tbody') {
+      newElement = document.createElement('td');
+    } else {
+      newElement = document.createElement('th');
+    }
+
+    newElement.textContent = line.children[1].textContent;
```

Comment:
Ensure that 'line.children[1]' exists and has the expected structure to avoid potential runtime errors. Consider adding a check or handling for cases where 'line.children[1]' might be undefined.